tools/fix_auth/fix_auth.rb

```diff
@@ -69,6 +69,10 @@ def fix_database_yml
       FixDatabaseYml.run({:hardcode => options[:password]}.merge(run_options))
     end
 
+    def load_rails
+      require File.expand_path("../../../config/application.rb", __FILE__)
```

Comment:
Don't you want to run the initializers? Typically, tools in the tools directory require config/environment to load rails, which is the same as above ^ but it also calls Vmdb::Application.initialize!

@kbrock I don't know that it matters but was curious since all the other tools load config/environment (since that's what I always do).